packages/calcite-components/src/components/carousel/carousel.tsx

```diff
@@ -512,13 +514,19 @@ export class Carousel
         break;
       case "Home":
         event.preventDefault();
+        if (this.selectedIndex === 0) {
```

Comment:
should this move before event.preventDefault so event isn't prevented if nothing happens?

Reply:
I think I'd still expect it to be prevented. Without it, the Home / End keys could navigate the Carousel with one keystroke and then unexpectedly scroll the page with the next.

Reply:
Yep. makes sense.